configs/squish.py

```diff
@@ -2,4 +2,4 @@
 
 AUT_PORT = 61500 + int(os.getenv('BUILD_NUMBER', 0))
 SERVER_PORT = 4322 + int(os.getenv('BUILD_NUMBER', 0))
-CURSOR_ANIMATION = False
+CURSOR_ANIMATION = True
```

Comment:
Is it really needed?

Reply:
Yes, you are right, just forgot to remove this change. Changed back to False